### PR TITLE
Rewrite control-slider

### LINKED ISSUE #188: Rewrite range component
### Criteria:

- TypeScript
- Functional component
- Radix inner workings. Here's the API Reference for range: https://www.radix-ui.com/docs/primitives/components/slider#custom-apis

### PR DESCRIPTION
**Pull request by tristen**
Changes to the control-slider component to use TypeScript, and React testing library, and the [slider primitives in Radix](https://www.radix-ui.com/docs/primitives/components/slider#custom-apis). Slider now supports active range styling and multiple thumbs!

🚨 **Dropped prop changes**:

- `value` should now be formatted as an array of numbers

✅  **Added prop changes**:

- `themeControlRangeActive` is a new prop that accepts classnames to style the active part of the track between a multirange slider.

---

Closes #188

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +17,6 @@ Changelog
   - Renamed `initialDisplayValue` to `displayValue`
   - Dropped 'themeControlFileClear'
 - [breaking] Removed clear implementation in `src/components/control-file`. Just a button element that triggers <input type="file" /> is implemented.
->>>>>>> e0fe270 (Add to changelog)
```
Comment (miafan23) [inline CHANGELOG.md:17]: Nice catch!

**Review by miafan23 — COMMENTED** [review]
Looks great! It's awesome the value supports array now.
I feel like it is a little hard to click the line due to its height. I checked radix-ui and found the height of the clickable area is not the line itself but the height of thumb. It would be nice if our component behaves the same.

Screen Shot 2022-08-25 at 9 50 56
Screen Shot 2022-08-25 at 9 51 04

**Comment by tristen** [comment]
@miafan23 nice catch on the catchment area : ) revised the CSS based on the feedback 👍

**Review by miafan23 — APPROVED** [review]
LGTM!

**Pull request merged**